### ISSUE TITLE
Elemental Revenants floating in the air in Wintergrasp

### ISSUE THREAD

**Issue by Xathal** [dalaranwow/dalaran-wow#5011]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Revenant mobs (greater elementals in Wintergrasp) floating in the air
**Expected behaviour**: Revenants stay on the ground/in the water (for water revenants)

**Steps to reproduce the problem**:

1. Go to Wintergrasp
2. Fly around the lakes and the Forest of Shadows
3. See the revenants flying towards the air and falling back
![wowscrnshot_110417_145239](https://user-images.githubusercontent.com/32802233/32406024-d672230a-c170-11e7-9d77-09615e8371cc.jpg)
![wowscrnshot_110417_145244](https://user-images.githubusercontent.com/32802233/32406025-d6a05f22-c170-11e7-868d-a66baa35320c.jpg)
![wowscrnshot_110417_145420](https://user-images.githubusercontent.com/32802233/32406026-d6c7517c-c170-11e7-8617-03acb8264775.jpg)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

http://www.wowhead.com/npc=30877/water-revenant

http://www.wowhead.com/npc=30872/shadow-revenant

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello there,

Thank you for your report, it has been already fixed and will be applied to Algalon with the next maintenance.
